### ISSUE TITLE
community benchmark run.sh changes

### ISSUE THREAD

**Issue by gareth-ellis** [nodejs/benchmarking#193]
I'm putting this here to cover a few changes I think are necessary to the community benchmark job in jenkins.

Currently the version we run is from the core-benchmarks branch in the benchmarking repo, I'll enter a PR to bring that into master and then update the job in jenkins to run from master.

Once done, I have a few more changes I want to apply to make things clearer in the job output.

1) I currently redirect build stdout to a file, but stderr still produces a lot of output which can make it trickier to understand what is happening. Suggest redirect stderr to a file too, check return code of build and then echo that out and exit if one build fails.

2) Currently i save the benchmark csvs in the workspace, i want to do the same for the build stdout and stderr. Probably need to look at a cleanup process too for this, 

3) Longer term plan i think would be to move these files off somewhere outside of the benchmark machine, perhaps to a folder under where the benchmarking.nodejs.org is kept, then remove after x amount of time. We've discussed this in another issue, but i've not done anything more with it since then.

@mhdawson , could I get another benchmark job created for testing these changes in the branch before we go and merge into master? I'll open a PR now for moving current changes into master, but once done I'd suggest I use the branch to stage changes and test them before merging into master where the job will be run from.

**Comment by mhdawson** [comment]
@gareth-ellis did I ever create that for you or are you still waiting on me(sorry I can't remember)? If so please ping and we'll get it setup.

If I did create one can you post on update on progress so far.
